File: DotNetAssignments/withDBEntity/company/Entity/Employee.cs
using System.ComponentModel.DataAnnotations.Schema;
using System;


namespace company.Entity
{
    [Table("EmployeeInfo")]
    public class Employee
    {
        [column("EmployeeId")]
        public int Id {get;set;}
        public string Name { get; set; }
        public double Salary {get;set;}
        public int DepartmentId {get;set;}
        public Department Dept {get;set;}        
    }
}

File: DSA/stackAndqueue/src/com/assignment/Employee.java
package com.assignment;

public class Employee {
	public int employeeId;
	public String employeeName;
	public char gender;
	public int salary;
	public Employee next;

	public Employee(int employeeId, String employeeName, char gender, int salary) {
		this.employeeId = employeeId;
		this.employeeName = employeeName;
		this.gender = gender;
		this.salary = salary;
		this.next = null;
	}
}
